views/partials/docs/moons/getAll.pug

```diff
@@ -0,0 +1,31 @@
+div(class="section")
+  div(class="lable") Method
+  div(class="value delete") get
+div(class="section")
+  div(class="lable") Route
+  div(class="value") http:localhost:2319/moons
```

Comment:
This route doesn't work

![image](https://user-images.githubusercontent.com/5775716/84069573-b9781800-a998-11ea-9092-4298eeebc7f5.png)